packages/frontend/src/features/sqlRunner/components/visualizations/barChartBizLogic.ts

```diff
@@ -0,0 +1,259 @@
+import { DimensionType, friendlyName, type ResultRow } from '@lightdash/common';
+import { getPivotedResults } from './usePivotedResults';
+
+type SqlColumn = {
+    id: string;
+    type: DimensionType;
```

Comment:
Not sure how we can get the `type` from current results endpoint. 
It returns a `ResultRow[]`, so that doesn't include the type.
So either the endpoint returns the results **and** the types for each, or we have to find this information somewhere else 🤔